requirements_dev.txt

```diff
@@ -14,5 +14,5 @@ scipy>=1.5.0
 pyfftw
 PyQt5
```

Comment:
What's the reason PyQt5 is not listed in setup.py?

Reply:
There's currently no active code importing it. I'll just remove this.

Reply:
removed